src/components/Process/Detail.tsx

```diff
@@ -0,0 +1,202 @@
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  CardProps,
+  Flex,
+  Grid,
+  GridItem,
+  Tab,
+  TabList,
+  TabPanel,
+  TabPanels,
+  Tabs,
+  Tag,
+  Text,
+  useBreakpointValue,
+  VStack,
+} from '@chakra-ui/react'
+import {
+  ElectionDescription,
+  ElectionHeader,
+  ElectionResults,
+  ElectionSchedule,
+  ElectionTitle,
+} from '@vocdoni/chakra-components'
+import { useElection } from '@vocdoni/react-providers'
+import { ElectionStatus, IElectionInfoResponse, PublishedElection } from '@vocdoni/sdk'
+import { FallbackHeaderImg } from '~constants'
+import { HeroHeaderLayout } from '~src/layout/HeroHeaderLayout'
+import { CopyButton, ReducedTextAndCopy } from '~components/CopyButton'
+import { Trans, useTranslation } from 'react-i18next'
+import { ElectionStatusBadge } from '~components/Organizations/StatusBadge'
+import { PropsWithChildren } from 'react'
+import { OrganizationCard } from '~components/Organizations/Card'
+import { RawContentBox } from '~src/layout/ShowRawButton'
+import { useElectionKeys } from '~queries/processes'
+import { ucfirst } from '~utils/strings'
+
+const Detail = () => {
+  const { election } = useElection()
+  const { t } = useTranslation()
+
+  if (!election || !(election instanceof PublishedElection)) return null
```

Comment:
The process rendering should take into account the rendering of both InvalidElection and ArchivedElection. So this should be something more like...

```suggestion
  if (!election) return null
  if (election instanceof InvalidElection) {
    return <InvalidElection />
  }
```

And have a InvalidElection component that should render at least the id and an alert box saying this election is invalid.

Reply:
624e418